kolibri/utils/options.py

```diff
@@ -271,6 +273,22 @@ def multiprocess_bool(value):
         return False
 
 
+def inherits_from_storage(value):
+    try:
+        modules = value.split(".")
+        klass = modules.pop()
+        module_path = ".".join(modules)
+        module = importlib.import_module(module_path)
+        Klass = getattr(module, klass)
+        return issubclass(Klass, Storage)
+    except ImportError:
+        logger.error("Default file storage is not available.")
+        raise VdtValueError(value)
+    except Exception:
```

Comment:
Shouldn't ever be catching a bare Exception, unless for very good reason - it can hide a multitude of sins.